### ISSUE TITLE
New Interface

### ISSUE THREAD

**Issue by flyingsonu122** [flyingsonu122/tarana#44]
1. **Make New Look Like This**

![Desktop - 1](https://user-images.githubusercontent.com/51280276/158061509-a15f891a-a845-4dd1-9256-56a6dafe3a1b.png)

![Desktop - 2](https://user-images.githubusercontent.com/51280276/158061529-53e41ccf-7ab2-48b6-9c39-55ba8f2aa0bb.png)

![Desktop - 3](https://user-images.githubusercontent.com/51280276/158061542-f6a13a02-9610-4e5e-b1ce-b420c538c391.png)

**Comment by tejal-1303** [comment]
I would like to work on this issue
I am a GSSOC contributor
Please assign me this task

**Comment by SwathiReddy03** [comment]
I am a GSSOC contributor
I also want to work on this issue 
as it is a big task we can both work on this task together 
what do you think?

**Comment by AmanRana10** [comment]
I can make the songs list look like this because i had previously worked on it as well and achieved the same...

**Comment by payalsamal** [comment]
I am GSSOC22 contributor and i wanna work on this new interface.

**Comment by flyingsonu122** [comment]
> I am GSSOC22 contributor and i wanna work on this new interface. i also wanna suggest to display movie name with artist and download option.

Alright please work on it @payalsamal

**Comment by flyingsonu122** [comment]
who is working on it, **first person to complete** will be considered

**Comment by Tanujkanti4441** [comment]
can i work on this

**Comment by flyingsonu122** [comment]
> can i work on this

Yes you should

**Comment by Tanujkanti4441** [comment]
hey @flyingsonu122 don't know why it is taking too much time and data for me to clone it which is not supposed to be happen because there are only html css and js files. Even i have deleted the music and image directory completely. Is it same for others also

**Comment by flyingsonu122** [comment]
do not clone it, download only html css and js files only ,make changes, and upload you files

**Comment by flyingsonu122** [comment]
> Hi @flyingsonu122 , Can I work on this for Hacktoberfest2022? I'll try my best

Yes pls,

**Comment by flyingsonu122** [comment]
> Hi @flyingsonu122 , Can I work on this for Hacktoberfest2022? I'll try my best

Yes pls,

**Comment by dyarawilliams** [comment]
Hi @flyingsonu122 So here's what I have. Hope I did well. But I was just curious about how you fetched the songs and got them to play in the app I hard-coded it for the UI. 


![tarana-draft](https://user-images.githubusercontent.com/15236104/194681070-5c11877a-2a01-42de-95fe-920c7f6e1e55.PNG)

**Comment by dyarawilliams** [comment]
Should I put in a pull request with the changes I have or would you like me to make more changes?

**Comment by flyingsonu122** [comment]
> Should I put in a pull request with the changes I have or would you like me to make more changes?

yes in [dynamic](https://github.com/flyingsonu122/tarana/tree/dynamic) branch
